### PR TITLE
minor Jira tracker related improvements

### PR DESCRIPTION
**Pull request by osoukup**
During the last sprint review preparations I had issues creating Jira trackers (so I used Bugzilla as an example). This is the followup fixing the stuff I had issues with. The first commit is actually not really a fix and it is opinionated. I am proposing the order but I am not pushing it. If you are against, we can drop it. Just in the original version it was hard to see the differences of my **.env** and the compose **environment** by human.

Closing OSIDB-1735

### CODE REVIEW

**Comment by osoukup** [comment]
@RedHatProductSecurity/osidb-devs please review

File: docker-compose.yml [hunk 1]
```diff
@@ -84,6 +84,7 @@ services:
         PRODUCT_DEF_URL: ${PRODUCT_DEF_URL}
         PS_CONSTANTS_URL: ${PS_CONSTANTS_URL}
         SNIPPET_CREATION: ${SNIPPET_CREATION}
+        TRACKERS_SYNC_TO_JIRA: ${TRACKERS_SYNC_TO_JIRA}
```
Comment (jobselko) [inline docker-compose.yml:87]: Should this be added into `osidb-ops` too?

**Review by jobselko — APPROVED** [review]
LGTM

File: docker-compose.yml (hunk 1 above)
Comment (osoukup) [reply docker-compose.yml:87]: As long as the default value of `False` is OK I think it is not needed. Anyway this commit only allows the previously non-existing option to generally set it via the environment but does not account for any specific environment. When we think it should be defined in some we can propose that elsewhere.

**Pull request merged**